contrib/world_builder/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
I think we had adjusted this to 3.13.4 because that is the minimum version for ASPECT